build.cake

```diff
@@ -412,108 +413,44 @@ Task("PackageChocolatey")
 	.Description("Creates chocolatey packages of the console runner")
```

Comment:
Does this task depend on the Build-task (since we specify files from the BIN_DIR directory?)